README.md

```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
 
```

Comment:
L3 should have a link for "Amazon Quantum Ledger Database (QLDB)" as Python has.

Reply:
Additionally, the badges should be right after the header on L2, and then the intro blurb should be after.

Reply:
Changed